### DISCUSSION TITLE
WT32-SC01 -80MHz SPI?

### DISCUSSION THREAD

**Discussion by FrankBoesing** [Bodmer/TFT_eSPI#1462] [Q&A - General] [answered]
Hi, a question:

I bought the WT32-SC01, which is board with ESP-Wrover and a ST7796 display.
The Datasheet (Pg. 6) says, it supports 80MHz. It is connected to the HSPI port.
Is there any additional config needed to use 80MHz? It works perfectly with 40MHz, but when I switch to 80, the display stays blank.

Link to "Datasheet" : http://www.wireless-tag.com/wp-content/uploads/2021/01/WT32-SC01DataSheetV3.3-2-with-nuts.pdf

Thank you in advance!

**Comment by Bodmer** [comment, marked as answer]
The WT32-SC01 display is wired to the HSPI pins. By default the TFT_eSPI library uses the VSPI port and these will get routed through an internal multiplexer which degrades performance.  To avoid the multiplexer [use this setting](https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setup.h#L335-L338) in the setup file.

Note that the ST7796 display is actually only rated at 16MHz maximum SPI clock, however most displays run at 40MHz. Some may run at 80MHz but this is 5x overclocking and it is likely to be unreliable.

**Reply by FrankBoesing** [reply]
Thank you!
Ok, this means the display does not support 80MHz, other than stated in the "Datasheet".

p.s. This library is great!
